cypress/elements/shared/button-selectors.js

```diff
@@ -1,5 +1,4 @@
 export const BUTTON_SELECTORS = {
-  back: '[data-test-id="app-header-back-button"]',
```

Comment:
Please remove all places where BUTTON_SELECTORS.back is used, and change to "cy.go('back')":
1. cypress/integration/configuration/permissions.js - line 55
2. cypress/integration/preorders/createAndEditVariantsInPreorder.js - line 78

Reply:
Wait, `app-header-back-button` did not trigger the browser's back button. Those two are not equivalent.

Reply:
I edited my comment